components/logins/android/src/test/java/mozilla/appservices/logins/DatabaseLoginsStorageTest.kt

```diff
@@ -212,30 +212,6 @@ class DatabaseLoginsStorageTest {
         finishAndClose(test)
     }
 
-    @Test
-    fun testMigrationMetrics() {
-        // We captured this string from one of the rust tests, then lightly
-        // edited it. Note that none of the "phases" will ever have data.
-        val json = """
-            {"fixup_phase":{
-                "num_processed":0,"num_succeeded":0,"num_failed":0,"total_duration":0,"errors":[]
-            },
-            "insert_phase":{"num_processed":0,"num_succeeded":0,"num_failed":0,"total_duration":0,"errors":[]
-            },
-            "num_processed":3,"num_succeeded":1,"num_failed":2,"total_duration":53,"errors":[
-                "Invalid login: Login has illegal field: Origin is Malformed",
-                "Invalid login: Origin is empty"
-            ]
-        }"""
-        recordMigrationMetrics(json)
-        assertEquals(3, LoginsStoreMetrics.migrationNumProcessed.testGetValue())
-        assertEquals(2, LoginsStoreMetrics.migrationNumFailed.testGetValue())
-        assertEquals(1, LoginsStoreMetrics.migrationNumSucceeded.testGetValue())
-        assertEquals(53L, LoginsStoreMetrics.migrationTotalDuration.testGetValue())
-        // Note the truncation of the first error string.
-        assertEquals(listOf("Invalid login: Login has illegal field: Origin is ", "Invalid login: Origin is empty"), LoginsStoreMetrics.migrationErrors.testGetValue())
-    }
-
```

Comment:
Should we remove this file entirely?  I don't think it's doing anything right now, but if we want to add storage tests I then maybe it makes sense to leave it around.  I'm really not sure.